src/models/Pet.js

```diff
@@ -0,0 +1,83 @@
+const mongoose = require('mongoose');
+
+const PetSchema = new mongoose.Schema(
+  {
+    petName: {
+      type: String,
+      required: [true, 'Please provide pet name'],
+      maxlength: [100, 'Name can not be more than 100 characters'],
+    },
+    petType: {
+      type: String,
+      enum: ['Cat', 'Dog', 'Other'],
+      default: 'Cat',
+      required: [true, 'Please provide type'],
+    },
+    date: {
+      type: Date,
+      default: Date.now,
+    },
+    description: {
+      type: String,
+      required: [true, 'Please provide product description'],
+      maxlength: [2000, 'Description can not be more than 2000 characters'],
+    },
+    image: {
+      type: String,
+      default: '/uploads/example.jpeg',
+    },
+    breed: {
+      type: String,
+      enum: [
+        'Domestic Short Hair',
+        'Domestic Medium Hair',
+        'Domestic Long Hair',
+      ],
+      default: 'Domestic Short Hair',
+    },
+    age: {
+      type: String,
+      enum: ['Kitten', 'Young', 'Adult', 'Senior'],
+      default: 'Adult',
+    },
```

Comment:
Since cats are not only pets who might be adopted I do not think 'kitten' is an appropriate choice. A cub could be a choice for a carnivorous mammal. You might have to do some research for a good word.